ConsumerAPI/NodeJs/main.js

```diff
@@ -7,11 +7,10 @@ const identity = {
     token: 'ZHVtbXlfdG9rZW4=',
 };
 
-const PROCESS_MODEL_ID= 'Lager-Manuell';
-//const PROCESS_MODEL_ID= 'Lager-Teilautomatisch';
+const PROCESS_MODEL_ID= 'Benutzeraktivierung';
 
-const START_EVENT_ID = 'VersandauftragErhalten';
-const END_EVENT_ID = 'VersandauftragVersendet';
+const START_EVENT_ID = 'StartAktivierung';
+const END_EVENT_ID = 'EndeAktivierung';
 
 function createConsumerClient(url) {
```

Comment:
Um das Ganze Deckungsgleich mit dem .NET Sample zu halten (und da die APIs demnächst eh umbenannt werden):

```suggestion
function createProcessEngineClient(url) {
```